src/picongpu/include/fields/background/templates/TWTS/BField.hpp

```diff
@@ -41,6 +41,14 @@ class BField
 public:
     typedef float_X float_T;
     
+    enum PolarizationType
+    {
+        /* The linear polarization of the TWTS laser is defined
+         * relative to the plane of the pulse front tilt. */
+        NORMAL_TO_TILTPLANE = 1u, /* use Ex-fields in TWTS laser internal coordinate system */
```

Comment:
That is where I got the inspiration from. However, I used different names, because the "in-plane" pulse albeit being linearly polarized is neither polarized in Y or Z direction. The normal direction however can be renamed to `LINEAR_X`.